workflows_core/workflow/helpers.py

```diff
@@ -37,6 +37,6 @@ def read_token_from_script():
     parser = argparse.ArgumentParser()
     parser.add_argument("token", help="The token used for the workflow config.")
     args = parser.parse_args()
-    token = args.token
+    token = args.workflow_token
```

Comment:
This won't work? You're parsing in `token` as a parameter but submitting `workflow_token`